Extension/src/LanguageServer/configurations.ts

```diff
@@ -1896,9 +1896,7 @@ export class CppProperties {
         }
 
         // Resolve and split any environment variables
-        paths = this.resolveAndSplit(paths, undefined, this.ExtendedEnvironment);
```

Comment:
Actually, we don't split the paths later down the function, so we still need to do it at some point.  We can do it in the validation loop since you'll be able to keep the original string there.